openbook_common/tests/helpers.py

```diff
@@ -23,6 +23,11 @@ def make_fake_post_text():
     return fake.text(max_nb_chars=settings.POST_MAX_LENGTH)
 
 
+def make_fake_username():
+    username = fake.text(max_nb_chars=settings.USERNAME_MAX_LENGTH)
```

Comment:
there is a `fake.user_name` function. Text collides many times in tests as its an `n` number of words

Reply:
I changed it -  though how do we ensure it stays within our username constraints.. i see everywhere in tests just a fake.user_name is used..